src/test/bin/quic_gtest.cpp

```diff
@@ -1878,6 +1878,17 @@ TEST(Misc, ClientDisconnect) {
     }
 }
 
+TEST(Misc, StatelessResetKey) {
+    TestLogger Logger("QuicTestStatelessResetKey");
+    if (TestingKernelMode) {
+        uint8_t Param = 1;
+        ASSERT_TRUE(DriverClient.Run(IOCTL_QUIC_RUN_STATELESS_RESETKEY, Param));
+    }
+    else {
+        QuicTestStatelessResetKey(true);
```

Comment:
If we're always going to set this to `true` then there isn't much reason to pass a parameter.